backend/src/test/java/gov/cdc/usds/simplereport/api/PatientUploadTest.java

```diff
@@ -0,0 +1,42 @@
+package gov.cdc.usds.simplereport.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.util.LinkedMultiValueMap;
+
+public class PatientUploadTest extends BaseApiTest {
+  public static final int PATIENT_PAGEOFFSET = 0;
+  public static final int PATIENT_PAGESIZE = 1000;
+
+  @Test
+  void uploadPatientCSV() throws Exception {
+    useSuperUser();
+    JsonNode response = uploadPatients();
+    assertEquals(
+        "\"Successfully uploaded 1 record(s)\"", response.get("uploadPatients").toString());
```

Comment:
There is an `assertLastAuditEntry` if you want to validate what was saved in the audit log, but that is a lot of copying and pasting.